### DISCUSSION TITLE
Release 3.0.0 ?

### DISCUSSION THREAD

**Discussion by nicolasiltis** [jupyter/enterprise_gateway#1096] [General]
Do you have an idea when you will release the version 3.0.0 ? (I'd like to get the new helm chart through the tarball) 

Thanks for the work on this project !

**Comment by kevin-bates** [comment]
Hi @nicolasiltis - thanks for opening this discussion.  

Since EG 3.0 is a major release, it's important we get any API-changing changes in that we can.  Of the currently outstanding pull requests, I think we should hold things for #1000 (cc: @telamonian).  That said, I also think we should produce some number of pre-releases (beta and at least one release candidate) to allow for early adopters, etc.

I will be out for several days, returning next Wednesday (May 31), so I don't expect anything to happen within the next week or so (sorry for the delay).
